lib/records/fhir.rb

```diff
@@ -182,19 +184,32 @@ def self.encounter(encounter, fhir_record, patient)
         encounter_data = ENCOUNTER_LOOKUP[encounter['type']]
         reason_data = COND_LOOKUP[encounter['reason']] if encounter['reason']
 
+        end_time = encounter['end_time'] || encounter['time'] + 15.minutes
+
         fhir_encounter = FHIR::Encounter.new('id' => resource_id,
                                              'status' => 'finished',
                                              'class' => { 'code' => encounter_data[:class] },
                                              'type' => [{ 'coding' => [{ 'code' => encounter_data[:codes]['SNOMED-CT'][0], 'system' => 'http://snomed.info/sct' }], 'text' => encounter_data[:description] }],
                                              'patient' => { 'reference' => patient.fullUrl.to_s },
-                                             'period' => { 'start' => convert_fhir_date_time(encounter['time'], 'time'), 'end' => convert_fhir_date_time(encounter['time'] + 15.minutes, 'time') })
+                                             'period' => { 'start' => convert_fhir_date_time(encounter['time'], 'time'), 'end' => convert_fhir_date_time(end_time, 'time') })
         if reason_data
           fhir_encounter.reason = FHIR::CodeableConcept.new('coding' => [{
                                                               'code' => reason_data[:codes]['SNOMED-CT'][0],
                                                               'display' => reason_data[:description],
                                                               'system' => 'http://snomed.info/sct'
                                                             }])
         end
+
+        if encounter['discharge']
+          fhir_encounter.hospitalization = FHIR::Hospitalization.new('dischargeDisposition' => {
```

Comment:
I think I'm confused... is `Hospitalization` a FHIR resource? Which version of FHIR?